### PR TITLE
Created test: User taken to ride details upon trip completion

### PR DESCRIPTION
**Pull request by jeremyauclert**
## JIRA
[MOB-2954](https://karhoo.atlassian.net/browse/MOB-2954

## DESCRIPTION
Created UI test for checks on trip completion from Trip status: POB

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [x] New feature 
- [x] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [ ] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: app/src/androidTest/java/com/karhoo/karhootraveller/splash/SplashTestsEndToEnd.kt [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    }
+
+    companion object {
+
+        private val origin = TestData.ORIGIN_TRIP
+        private val destination = TestData.DESTINATION_TRIP
+        private val initialTripDetails = TripInfo(origin = origin, destination = destination)
+        private val identicalAddresses = TripInfo(origin = origin, destination = origin)
+
+//        val CLEAN_TRIP_INTENT = Intent().apply {
```
Comment (jsm20) [inline app/src/androidTest/java/com/karhoo/karhootraveller/splash/SplashTestsEndToEnd.kt:191]: Is this needed later or can it be deleted?

File: app/src/androidTest/java/com/karhoo/karhootraveller/splash/SplashTestsEndToEnd.kt [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        val sharedPreferences = context.getSharedPreferences("user", Context.MODE_PRIVATE)
+        val editor = sharedPreferences.edit()
+        sharedPreferences.edit()
+                .putString("first_name", "John")
+                .putString("last_name", "Smith")
+                .putString("email", "test@test.test")
+                .putString("mobile_number", "123")
+                .putString("user_id", "1234")
+                .putString("organisations", Gson().toJson(
+                        listOf(Organisation(id = "organisation_id",
```
Comment (jsm20) [inline app/src/androidTest/java/com/karhoo/karhootraveller/splash/SplashTestsEndToEnd.kt:93]: Might be worth putting the Organisation in ServerTestData

File: app/src/androidTest/java/com/karhoo/karhootraveller/splash/SplashTestsEndToEnd.kt (hunk 1 above)
Comment (jeremyauclert) [reply app/src/androidTest/java/com/karhoo/karhootraveller/splash/SplashTestsEndToEnd.kt:191]: Oh sugar, I forgot to take it out. I'll do that now

Comment (jeremyauclert) [reply app/src/androidTest/java/com/karhoo/karhootraveller/splash/SplashTestsEndToEnd.kt:191]: It's from my other branch and not stashing

**Review by jsm20 — APPROVED** [review]

**Pull request merged**